docs/analyzer/getting-started.md

```diff
@@ -331,18 +331,19 @@ For all supported syntax, please check the [fixtures](https://github.com/open-wc
 
 ### Supported JSDoc
 
-| JSDoc                         | Description                                        |
-| ----------------------------- | -------------------------------------------------- |
-| `@attr`, <br>`@attribute`     | Documents attributes for your custom element       |
-| `@prop`, <br>`@property `     | Documents properties for your custom element       |
-| `@part`,<br>`@csspart`        | Documents your custom elements Shadow Parts        |
-| `@slot`                       | Documents the Slots used in your components        |
-| `@cssprop`,<br>`@cssproperty` | Documents CSS Custom Properties for your component |
-| `@fires`,<br>`@event`         | Documents events that your component might fire    |
-| `@tag`,<br>`@tagname`         | Documents the name of your custom element          |
-| `@summary`                    | Documents a short summary                          |
-| `@internal`,<br>`@ignore`     | To omit documentation of internal details          |
-| `@default`                    | Documents the default value for a property         |
+| JSDoc                         | Description                                                                       |
+| ----------------------------- | --------------------------------------------------------------------------------- |
+| `@attr`, <br>`@attribute`     | Documents attributes for your custom element                                      |
+| `@prop`, <br>`@property `     | Documents properties for your custom element                                      |
+| `@part`,<br>`@csspart`        | Documents your custom elements Shadow Parts                                       |
+| `@slot`                       | Documents the Slots used in your components                                       |
+| `@cssprop`,<br>`@cssproperty` | Documents CSS Custom Properties for your component                                |
+| `@cssState`                   | Documents the custom CSS state of your custom element when using ElementInternals |
```

Comment:
I'm a little confused, the code description for `@cssState` describes it as the "custom CSS state of your custom element when using ElementInternals".

After looking at the [API in MDN](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/states), there's no mention about CSS, the language, only CustomStateSet.

Is this part of the CSS spec and MDN just isn't clear about this, or is it just confusing that both abbreviations are `css`, if so, would it make sense to use a different abbreviation, or clarify this?

Let me know if I've misunderstood something :)

This is a cool change and I didn't know this ElementInternals feature existed, so this is more about me learning about new features than a criticism of the API design

Reply:
Ignore me, I read the changes [here](https://github.com/webcomponents/custom-elements-manifest/pull/128) and it cleared things up :)

Still cool to hear about this feature! Awesome